ecosystem/theme-default/src/client/styles/code.scss

```diff
@@ -264,3 +264,13 @@ div[class*='language-'] {
     }
   }
 }
+
+// ===============================
+
+tr {
+  &:nth-child(2n) {
+    code {
+      background-color: var(--c-bg-dark);
+    }
+  }
+}
```

Comment:
Maybe we should put them to the table styles file.

Reply:
Done at commit https://github.com/vuepress/vuepress-next/pull/1131/commits/9045a4f8303eec5b48fdbb6802ef0d8ba7042dc0